_data/hosting.yml

```
@@ -436,8 +436,7 @@ websites:
       tfa: Yes
       sms: Yes
       software: Yes
-      hardware: Yes
-      doc: https://www.transip.nl/vragen/574-hoe-stel-two-factor-authentication/
+      doc: https://www.transip.nl/knowledgebase/artikel/162
 
```

Based on their doc for [logging in to their STACK product](https://www.transip.nl/knowledgebase/artikel/234-de-webinterface-van-stack/) and for [setting up 2FA on STACK](https://www.transip.nl/knowledgebase/artikel/1531-two-factor-authentication-2fa-voor-stack/), I think we should add an exception that enabling 2FA on one doesn't enable it on the other. This is similar to how we handle Andrews & Arnold and Hostiso.
```suggestion
      exceptions:
          text: "While both the main control panel and STACK support 2FA, each system is separate and requires its own setup."
```